### PR TITLE
OCD-4136

### PR DESCRIPTION
**Pull request by mattystank**

### CODE REVIEW

File: src/app/pages/listing/listing.jsx [hunk 1]
```diff
@@ -78,7 +78,7 @@ const useStyles = makeStyles({
   },
   pageHeader: {
     padding: '32px 0',
-    backgroundColor: palette.white,
+    backgroundColor: `${palette.white}`,
```
Comment (andlar) [inline src/app/pages/listing/listing.jsx]: There are 30+ other places in our code where we _don't_ encircle a simple palette.color with the interpolation brackets, including in some other classes in this file. Why did you do it here?

File: src/app/pages/listing/listing.jsx [hunk 2]
```diff
@@ -361,7 +365,7 @@ function ChplListingPage({ id, panel }) {
                    <span id="sed" className="page-anchor" />
                  </span>
                  <Box className={classes.sectionHeader}>
-                   <Typography variant="h2">Safety Enhanced Design (SED)</Typography>
+                   <Typography className={classes.sectionHeaderText} variant="h2">Safety Enhanced Design (SED)</Typography>
```
Comment (andlar) [inline src/app/pages/listing/listing.jsx:368]: The first few headers have this line all on one line, and the last few have it broken into three. I don't care too much which way we go, but it should be consistent

**Review by andlar — CHANGES_REQUESTED** [review]

File: src/app/pages/listing/listing.jsx (hunk 1 above)
Comment (mattystank) [reply src/app/pages/listing/listing.jsx]: I thought this was the standard practice with our palette imports, I will change it back to the previous version.

File: src/app/components/listing/details/compliance/direct-reviews.jsx [hunk 3]
```diff
@@ -64,6 +65,10 @@ const useStyles = makeStyles({
       flexDirection: 'row',
     },
   },
+  errorChip: {
+    color: `${palette.white}`,
+    backgroundColor: `${palette.error}`,
```
Comment (andlar) [inline src/app/components/listing/details/compliance/direct-reviews.jsx]: These should be un-interpolated as well. It's necessary to do this if there are multiple values in this row, like on line 52, where the `backgroundColor` has both a color and the !important tag, but if it's not needed, we shouldn't do it

Comment (mattystank) [reply src/app/components/listing/details/compliance/direct-reviews.jsx]: Thanks for the clarification!

**Pull request merged**